ban/core/resource.py

```
@@ -126,25 +128,54 @@ def as_version(self):
         """Resources plus relations references and metadata."""
         return self.serialize({f: {} for f in self.versioned_fields})
 
+    @property
+    def status(self):
+        return 'deleted' if self.deleted_at else 'active'
+
+    @classmethod
+    def active(cls):
+        return cls.select().where(cls.deleted_at.is_null())
+
+    def mark_deleted(self):
+        if self.deleted_at:
+            raise ValueError('Resource already marked as deleted')
```

No strong argument.
You'd prefer to silently return in that case?